chipsec/modules/tools/secureboot/te.py

```diff
@@ -355,94 +354,103 @@ def produce_te(fname, outfname):
     with open(fname, 'rb') as f:
         data = f.read()
     te_data = replace_header(data)
-    if (te_data is None):
+    if te_data is None:
         return 0
     with open(outfname, 'wb') as fte:
         fte.write(te_data)
     return 1
 
 def replace_efi_binary(orig_efi_binary, new_efi_binary):
-    logger().log( "[*] replacing EFI binary '{}'..".format(orig_efi_binary) )
+    logger().log("[*] replacing EFI binary '{}'..".format(orig_efi_binary))
     te_binary = new_efi_binary + '.te'
-    if not os.path.exists(te_binary): produce_te(new_efi_binary, te_binary)
+    if not os.path.exists(te_binary):
+        produce_te(new_efi_binary, te_binary)
     # back up original binary
     backup = orig_efi_binary + '.bak'
-    if not os.path.exists(backup): os.rename(orig_efi_binary, backup)
+    if not os.path.exists(backup):
+        os.rename(orig_efi_binary, backup)
     try:
         shutil.copy(te_binary, orig_efi_binary)
     except OSError as err:
-        logger().error( 'Cannot replace binary ({})'.format(err) )
+        logger().log_error('Cannot replace binary ({})'.format(err))
         return False
     return True
 
 def umount(drive):
     import subprocess
     if os.path.exists(drive):
-        res = subprocess.call( ["mountvol.exe", drive, "/D"] )
-        if res != 0: logger().warn( "Cannot unmount EFI System partition: {:d}".format(res) )
+        res = subprocess.call(["mountvol.exe", drive, "/D"])
+        if res != 0:
+            logger().warn("Cannot unmount EFI System partition: {:d}".format(res))
 
 def get_efi_mount():
     import subprocess
     for l in range(ord('z'), ord('a'), -1):
         if not os.path.exists('%c:\\' % l):
-            res = subprocess.call( ["mountvol.exe", "%c:\\" % l, "/S"] )
+            res = subprocess.call(["mountvol.exe", "%c:\\" % l, "/S"])
             if res != 0:
-                logger().error( "Cannot mount EFI System partition (status = {:d})".format(res) )
+                logger().log_error("Cannot mount EFI System partition (status = {:d})".format(res))
                 return None
             return '%c:\\' % l
-    logger().error( "Cannot mount EFI System partition. No drive letters to use." )
+    logger().log_error("Cannot mount EFI System partition. No drive letters to use.")
     return None
 
-def get_bootloader_paths( cfg_file ):
+def get_bootloader_paths(cfg_file):
     bootloader_paths = []
-    fcfg = open( cfg_file, 'r' )
-    logger().log( "[*] reading paths from '{}'..".format(cfg_file) )
+    fcfg = open(cfg_file, 'r')
```

Comment:
```suggestion
    with open(cfg_file, 'r') as fcfg:
```
and correct indenting in the following lines.